contrib/auto-completion/snippets/emacs-lisp-mode/.yas-parents

```diff
@@ -0,0 +1 @@
+prog-mode
```

Comment:
So do you think it's a bad idea? D: I think I could take it out without it having implications